crates/html-macro/src/parser/mod.rs

```diff
@@ -207,14 +208,19 @@ impl HtmlParser {
     ///
     /// html! { <div> { some_var_in_braces } </div>
     /// html! { <div> { some_other_variable } </div>
-    fn push_iterable_nodes(&mut self, stmt: &Stmt) {
+    fn push_iterable_nodes(&mut self, stmt: &Stmt, tokens: Option<proc_macro2::TokenStream>) {
```

Comment:
One way to improve this would be to use an enum instead of having multiple function arguments but only using one of them

```rs
enum NodesToPush<'a> {
  Stmt: &'a Stmt,
  Tokens: proc_macro2::TokenStream
}

push_iterable_nodes(&mut self, nodes: NodesToPush) {
  ...
   match nodes {
       NodesToPush::Stmt => {...}
       NodesToPush::Tokens => { ... }
   }
}
```